libs/src/userStateManager.js

```diff
@@ -11,10 +11,6 @@ class UserStateManager {
   constructor () {
     // Should reflect the same fields as discovery node's /users?handle=<handle>
     this.currentUser = null
-
```

Comment:
?

Reply:
looks like this was removed in master: https://github.com/AudiusProject/audius-protocol/blob/master/libs/src/userStateManager.js#L11-L13